yb-voyager/src/srcdb/source.go

```diff
@@ -205,6 +205,8 @@ func (source *Source) PopulateOra2pgConfigFile(configFilePath string) {
 			lines[i] = "PG_INTEGER_TYPE 1" // Required otherwise MySQL autoincrement sequences don't export
 		} else if strings.HasPrefix(line, "DEFAULT_NUMERIC") {
 			lines[i] = "DEFAULT_NUMERIC numeric"
+		} else if strings.HasPrefix(line, "DISABLE_PARTITION") {
+			lines[i] = "DISABLE_PARTITION 0"
```

Comment:
Why not set this directly to `1`?

Reply:
Disabling partitions while exporting schema would completely disable the partition-based skeleton that ora2pg exports by default. The current logic is to keep the skeleton and populate using a flat data table; we only lose out on partition-specific validation rather than partitions as a whole